org.eclipse.jdt.ui/core extension/org/eclipse/jdt/internal/corext/callhierarchy/CallHierarchy.java

```diff
@@ -37,10 +37,14 @@
 import org.eclipse.jdt.internal.ui.util.StringMatcher;
 
 public class CallHierarchy {
-    private static final String PREF_USE_IMPLEMENTORS= "PREF_USE_IMPLEMENTORS"; //$NON-NLS-1$
+
+	private static final String PREF_SHOW_ALL_CODE = "PREF_SHOW_ALL_CODE";	//$NON-NLS-1$
```

Comment:
Why do these constants copy the values from CallHierarchyCore.java? Is it important that the values are the same? Then please reuse it (declare package x-friend).

https://github.com/eclipse-jdt/eclipse.jdt.ui/pull/1687/files#diff-9d62c5f4c6ca62f274aa1583e7e4f95060651e52e840b380013e205f1d67a21aR49?

Reply:
In CallHierarchyCore I changed the three constants which I added to public and will create another PR for the three constants which have already been there when I started the issue and are also private. 
In the other PR I will make them Public and change the code in CallHierarchy.

Reply:
https://github.com/eclipse-jdt/eclipse.jdt.ui/pull/1706